lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -455,7 +479,8 @@ ENT:AddHook("Think","teleport",function(self,delta)
 		target=self:GetTargetAlpha()
 		self:SetData("alphatarget",target)
 	end
-	alpha=math.Approach(alpha,target,delta*66*self.metadata.Exterior.Teleport.SequenceSpeed)
+	local sequencespeed = self.metadata.Exterior.Teleport.SequenceSpeed * (self:GetData("demat-fast",false) and 1.1 or 1)
```

Comment:
Can you explain the purpose behind this change? Seems a little weird - is it still required?

Reply:
Speeding the sequence up ends demat earlier and allows for remat to begin faster. Could do a separate speed for the fast sequence if desired

Reply:
Yeah please put it into the metadata as SequenceSpeedFast or something